docs/topology/docker-compose.yml

```diff
@@ -0,0 +1,175 @@
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+
+  redis-node-3:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    networks:
+      cluster_subnet:
+          ipv4_address: 172.16.1.5
+    volumes:
+      - redis-cluster_data-3:/bitnami/redis/data
+    environment:
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+
+  redis-node-4:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    networks:
+      cluster_subnet:
+        ipv4_address: 172.16.1.6
+    volumes:
+      - redis-cluster_data-4:/bitnami/redis/data
+    environment:
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+
+  redis-node-5:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    networks:
+      cluster_subnet:
+        ipv4_address: 172.16.1.7
+    volumes:
+      - redis-cluster_data-5:/bitnami/redis/data
+    environment:
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+
+  redis-cluster-init:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    networks:
+      cluster_subnet:
+        ipv4_address: 172.16.1.8
+    depends_on:
+      - redis-node-0
+      - redis-node-1
+      - redis-node-2
+      - redis-node-3
+      - redis-node-4
+      - redis-node-5
+    environment:
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_CLUSTER_REPLICAS=1'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+      - 'REDIS_CLUSTER_CREATOR=yes'
+
+  shotover-0:
+    restart: always
+    depends_on:
+      - redis-node-3
+    image: shotover/shotover-proxy
+    network_mode: "service:redis-node-0"
+    volumes:
+      - type: bind
+        source: $PWD
+        target: /config
+
+  shotover-1:
+    restart: always
+    depends_on:
+      - redis-node-3
+    image: shotover/shotover-proxy
+    network_mode: "service:redis-node-1"
+    volumes:
+      - type: bind
+        source: $PWD
+        target: /config
+
+  shotover-2:
+    restart: always
+    depends_on:
+      - redis-node-3
+    image: shotover/shotover-proxy
+    network_mode: "service:redis-node-2"
+    volumes:
+      - type: bind
+        source: $PWD
+        target: /config
+
+  shotover-3:
+    restart: always
+    depends_on:
+      - redis-node-3
+    image: shotover/shotover-proxy
+    network_mode: "service:redis-node-3"
+    volumes:
+      - type: bind
+        source: $PWD
+        target: /config
+
+  shotover-4:
+    restart: always
+    depends_on:
+      - redis-node-4
+    image: shotover/shotover-proxy
+    network_mode: "service:redis-node-4"
+    volumes:
+      - type: bind
+        source: $PWD
+        target: /config
+
+  shotover-5:
+    restart: always
+    depends_on:
+      - redis-node-4
```

Comment:
redis-node-4 on shotover-5.

And maybe we should just have 3 instances of each to keep things simpler.